lib/puppet/application/cert.rb

```diff
@@ -132,7 +148,10 @@ def help
   List outstanding certificate requests. If '--all' is specified, signed
   certificates are also listed, prefixed by '+', and revoked or invalid
   certificates are prefixed by '-' (the verification outcome is printed
-  in parenthesis).
+  in parenthesis). If '--human-readable' or '-H' is specified listed
```

Comment:
There should have been a comma there.

"If '--human-readable', or '-H' is specified
listed certificates are formatted in a way..."

But "listed certificates" is a verbose way to say it anyways, so I've added a comma to the respective clauses (here and where we discuss machine formatted output) as well as removed "listed".